github/apps_hooks_test.go

```diff
@@ -0,0 +1,98 @@
+package github
```

Comment:
```suggestion
// Copyright 2021 The go-github AUTHORS. All rights reserved.
//
// Use of this source code is governed by a BSD-style
// license that can be found in the LICENSE file.

package github
```

No need for action... I'll add this required header.